src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -280,24 +269,22 @@ private void fireAuditFinished() {
      * @noinspectionreason ProhibitedExceptionThrown - There is no other way to
      *      deliver filename that was under processing.
      */
-    // -@cs[CyclomaticComplexity] no easy way to split this logic of processing the file
```

Comment:
SOC, SRP, feature envy -> `Functions are supposed to do "one thing", do it well, and do it only.`: https://cleancoders.com/episode/clean-code-episode-3